public/app/features/trails/MetricSelect/MetricSelectScene.tsx

```diff
@@ -399,6 +488,21 @@ export class MetricSelectScene extends SceneObjectBase<MetricSelectSceneState> {
               suffix={metricNamesWarningIcon}
             />
           </Field>
+          <Field label={'View by'} className={styles.displayOption}>
+            <Select
+              onChange={model.onPrefixFilterChange}
+              value={metricPrefix}
+              onOpenMenu={() => model.reportPrefixFilterInteraction(true)}
+              onCloseMenu={() => model.reportPrefixFilterInteraction(false)}
+              options={[
+                {
+                  label: 'All metric names',
+                  value: METRIC_PREFIX_ALL,
+                },
+                ...Array.from(rootGroup?.groups.keys() ?? []).map((g) => ({ label: `${g}_`, value: g })),
+              ]}
+            />
```

Comment:
Can we add a tooltip icon here to describe what we are viewing by? Something like "View by..." then some summary of this prefix docs information for metric naming convention and [a link to the docs on prefixes](https://prometheus.io/docs/practices/naming/#metric-names)

```
...should have a (single-word) application prefix relevant to the domain the metric belongs to. The prefix is sometimes referred to as namespace by client libraries. For metrics specific to an application, the prefix is usually the application name itself. Sometimes, however, metrics are more generic, like standardized metrics exported by client libraries. Examples:
  prometheus_notifications_total (specific to the Prometheus server)
  process_cpu_seconds_total (exported by many client libraries)
  http_request_duration_seconds (for all HTTP requests)
```

Reply:
I didn't know there was actually documentation on metric prefixes 😆 . I'm okay with this to add some more context. I believe we have to do some hacky things to make a tooltip work with a link on hover. But I feel like just text is enough?

> View by the metric prefix. A metric prefix is a single word at the beginning of the metric name, relevant to the domain the metric belongs to.

<img width="483" alt="Screenshot 2024-07-19 at 15 49 08" src="https://github.com/user-attachments/assets/541a518f-d364-4088-96e3-c2d6d29a4d96">

Reply:
https://github.com/grafana/grafana/pull/89481/commits/f1957c57dc284a76caccb0e1cf0885c308ae2515